### PR TITLE
addgoaltask view update

### PR DESCRIPTION
**Pull request by Jermaine-rbg**
added a post request to the AddGoalTask(View)

### CODE REVIEW

**Review by Bronji — APPROVED** [review]

File: TheDreamProject/vision_app/views.py [hunk 1]
```diff
@@ -70,7 +70,13 @@ def get(self, request):
         
         return render(request, 'addgoaltask.html',{
                 '   Goal_Form' : Goal_Form,
-            })       
+            })   
+
+    def post(self,request):
+        '''POST the data in the form submitted by the user, creating a new task in the todo list'''
+        Goal_Form = GoalForm(request.POST)
+        Goal_Form.save()
```
Comment (pab2163) [inline TheDreamProject/vision_app/views.py:78]:
```suggestion
        if Goal_Form.is_valid():
            Goal_Form.save()
```

**Review by pab2163 — CHANGES_REQUESTED** [review]
I added a small suggestion here based on getting the error from the screenshot when trying to submit a vision. I think you'll need to validate the data from the form before saving 

image

File: TheDreamProject/vision_app/views.py (hunk 1 above)
Comment (pab2163) [reply TheDreamProject/vision_app/views.py:78]: I'm still a little unclear on how this works more generally -- it seems strange that a `Goal_Form` is being submitted when I had just clicked submit to create a _vision_? Any idea why this is?

Comment (Jermaine-rbg) [reply TheDreamProject/vision_app/views.py:78]: When the user creates a vision using the Vision_Form it suppose to create that object in the database Vision model. After the data has been submitted the vision_name is rendered to the addgoaltask.html. 

Everything looks good for as models, forms and django forms. I might be missing something.

**Pull request closed without merging**